terraform/environments/cooker/test-ec2/main.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.16"
+    }
+  }
+
+  required_version = ">= 1.2.0"
+}
+
+resource "aws_instance" "app_server" {
```

Comment:
we don't create infra resource such as EC2 in this repo, this should go into here https://github.com/ministryofjustice/modernisation-platform-environments/tree/main/terraform/environments/cooker if you want to play around